Assets/HotmaxEditorVR/Hittables/propTile.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class propTile : MonoBehaviour, IHittable
+{
+    //--local refs
+    private stateManager _stateManagerMutatorRef;
+
+    public int currentPage;
+
+    public GameObject spawnableProp;
```

Comment:
I would make a note above these vars that says `FAUX CONSTRUCTOR - properties passed on prefab instantiation`

Reply:
got it